parm/soca/letkf/letkf_stage.yaml.j2

```diff
@@ -0,0 +1,66 @@
+######################################
+# set some variables
+######################################
+{% set gPDY = previous_cycle | to_YMD %}
+{% set gcyc = previous_cycle | strftime("%H") %}
+{% set PDY = current_cycle | to_YMD %}
+{% set cyc = current_cycle | strftime("%H") %}
+######################################
+# create working directories
+######################################
+mkdir:
+- "{{ DATA }}/Data"
+- "{{ DATA }}/obs"
+- "{{ DATA }}/INPUT"
+{% for mem in range(1, NMEM_ENS + 1) %}
+- "{{ DATA }}/ens/mem{{ '%03d' % mem }}"
+{% endfor %}
+copy:
+######################################
+# copy mom input template 
+######################################
+- ["{{ PARMgfs }}/gdas/soca/fms/input.nml", "{{ DATA }}/mom_input.nml.tmpl"]
+######################################
+# copy ensemble background files
+######################################
+{% for mem in range(1, NMEM_ENS + 1) %}
+# define variables
+# Declare a dict of search and replace terms to GDUMP on each template
+{% set tmpl_dict = {'ROTDIR':ROTDIR,
+                    'GDUMP_ENS':GDUMP_ENS,
```

Comment:
is GDUMP_ENS needed here?

Reply:
Not necessarily, but why not distinguish it from the deterministic?

Reply:
I'm referring to the find/replace in the tmpl_dict, is `GDUMP_ENS` a key that is found in the templates or is it jus `RUN`?

Reply:
One of them can go - I thought `RUN` is deprecated?

Reply:
What does the template have in it? Is it GDUMP_ENS or is it RUN (genuinely not sure but I can check)

Reply:
`declare -rx 'COM_OCEAN_HISTORY_TMPL=${ROTDIR}/${RUN}.${YMD}/${HH}/${MEMDIR}/model_data/ocean/history'`

Reply:
Ah, that's right, that's why RUN is in there

Reply:
So can we remove GDUMP_ENS from that key/value pair?

Reply:
I must be misunderstanding. Is this not only to find/replace things in the COM_ templates? Is there another use for the `tmpl_dict`?

Reply:
Ah, I think you're right, my mistake - somehow I thought it applied outside that as well